src/compensations/Compensations.tsx

```diff
@@ -93,13 +94,10 @@ const Compensations = ({ compensations, locations }: CompensationsProps) => {
           ) : null}
         </>
       )}
-      <div className={styles.benefits}>
-        {compensations.benefitsByLocation.map((benefit) => (
-          <div key={benefit._key} className={styles.benefitWrapper}>
-            {/* TODO: display benefits based on locations */}
-          </div>
-        ))}
-      </div>
+      <BenefitsByLocation
```

Comment:
Could we move the location filtering to `<Compensations />`, so that `<BenefitsByLocation />` (which could be renamed to `<Benefits />`) only receives the relevant benefits?